spark-jobs/src/main/scala/za/co/absa/enceladus/conformance/DynamicConformanceJob.scala

```diff
@@ -119,6 +119,8 @@ object DynamicConformanceJob {
       postProcessingService.onSaveOutput(conformedDf) // all enabled postProcessors will be run with the std df
     } finally {
 
+      spark.sparkContext.applicationAttemptId
+        .foreach(attemptId => Atum.setAdditionalInfo("conf_spark_attempt_id" -> attemptId))
```

Comment:
You can put this inside `PerformanceMetricTools.addJobInfoToAtumMetadata(...)` and use the metadata prefix (std/conf).
If Spark Session is not passed to `addJobInfoToAtumMetadata()`, you can extend this method with `attemptId: String`.